packages/debugger/src/panels/callstack/index.ts

```diff
@@ -81,6 +81,15 @@ export class Callstack extends PanelWithToolbar {
       })
     );
 
+    this.toolbar.addItem(
```

Comment:
I would prefer the button to appear on the breakpoints panel not on the callstack one.